DfE.FindInformationAcademiesTrusts.Data.AcademiesDb/Contexts/AcademiesDbContext.cs

```diff
@@ -16,13 +16,14 @@ public interface IAcademiesDbContext
     DbSet<GiasGroupLink> GiasGroupLinks { get; }
     DbSet<GiasGroup> Groups { get; }
     DbSet<MstrTrust> MstrTrusts { get; }
+
     DbSet<CdmAccount> CdmAccounts { get; }
-    DbSet<MisEstablishment> MisEstablishments { get; }
-    DbSet<MisFurtherEducationEstablishment> MisFurtherEducationEstablishments { get; }
     DbSet<CdmSystemuser> CdmSystemusers { get; }
     DbSet<TadTrustGovernance> TadTrustGovernances { get; }
     DbSet<ApplicationEvent> ApplicationEvents { get; }
     DbSet<ApplicationSetting> ApplicationSettings { get; }
+    DbSet<EstablishmentFiat> EstablishmentsFiat { get; }
+    DbSet<FurtherEducationEstablishmentFiat> FurtherEducationEstablishmentsFiat { get; }
```

Comment:
I think it would be good to have the schema in the class name and property name (like all the others) so we can tell at a glance where this particular type of establishment comes from. Without the schema name it sounds like this is FIAT's core interpretation of what an establishment is.

I.e. `MisMstrEstablishmentFiat` to follow the same convention as `GiasEstablishment`